src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 4;
```

Comment:
It takes a lot of time to get counter logic this way.
Try starting it from 0 and work from there.